<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>950</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>1801</width>
     <height>781</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="transformationAnchor">
    <enum>QGraphicsView::AnchorViewCenter</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>800</y>
     <width>721</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="b_m1">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>m1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_m3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>m3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_m5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>m5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_m15">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>m15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_h1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>H1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_h4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>H4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_d1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>1D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_w1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_mn1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>800</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>800</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>350</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="smooth">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>830</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
